dashboard/app/views/levels/editors/_dsl.html.haml

```diff
@@ -34,10 +34,10 @@
     %p Edit Markdown:
 
     %textarea#level_markdown_textarea
-    #markdown-preview
+    #level_markdown_textarea_preview
 
   .markdown.teacher
     %p Edit Teacher Only Markdown:
 
     %textarea#level_teacher_markdown_textarea
-    #teacher-markdown-preview
+    #level_teacher_markdown_textarea_preview
```

Comment:
Is there a reason for this beyond consistency? The reason I went with snake_case is because everything in _applab.html.haml seemed to use snake_case, and I wanted to consistently use `_preview` instead of `-preview`